### DISCUSSION TITLE
Support for VLC Add-On in HA

### DISCUSSION THREAD

**Discussion by Surgical71** [music-assistant/support#1234] [Feature requests and ideas]
Hi there

I am currently unable to setup Mass V2 with the VLC Add-On. This use to work in V1. Don't know if this is a know issue and still needs to be added or if I am just doing something wrong.

![image](https://user-images.githubusercontent.com/80483431/235345926-ba1faa1c-5b9d-490d-9e8c-6396a27884f0.png)

Regards

**Comment by OzGav** [comment]
I don’t know what the VLC addon does but there are only a limited number of native players currently in V2 as you have shown above. When the HA integration is written in a few weeks then all existing HA media players will be exposed.

**Reply by Surgical71** [reply]
> I don’t know what the VLC addon does but there are only a limited number of native players currently in V2 as you have shown above. When the HA integration is written in a few weeks then all existing HA media players will be exposed.

Thank you. I will keep this discussion open and test and close once the HA Integration is available.

**Comment by OzGav** [comment]
The Integration allows access to all HA players

**Reply by LeonvanHeerden** [reply]
I have Setup the VLC Telnet player on HA. Tested that it works through HA. 
Then I added the MA / HA integration and could eventually play through the VLC and Kodi speakers at the same time. (Once I could get the speaker to select.) I am adding an issue about the speaker selection that is not working always.

Thank you very much for the entire MA solution.

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
